pkg/orchestration/wiring/ec2compute/common/common.go

```diff
@@ -140,7 +140,12 @@ func WireUp(microServiceNameInSpec, ec2ComputePlanName string, stateResource *or
 	for _, dependency := range dependencies {
 		bound := false
 		if bindableShape, ok := dependency.Contract.FindShape(knownshapes.BindableEnvironmentVariablesShape); ok {
-			resourceReference := bindableShape.(*knownshapes.BindableEnvironmentVariables).Data.ServiceInstanceName
+			typedBindableShape := &knownshapes.BindableEnvironmentVariables{}
```

Comment:
This code belongs to knownshapes package, so it is easier for clients to use it. Instead of 

`dependency.Contract.FindShape...` 

I'd prefer

`bindableShapeData := knownshapes.FindBindableEnvironmentVariablesShape(dependency.Contract)`

`FindBindableEnvironmentVariablesShape` should return `BindableEnvironmentVariablesDate`